### ISSUE TITLE
Configuration Election Page

### ISSUE THREAD

**Issue by ddfridley** [EnCiv/undebate-ssp#23]
```
<ConfigurationElection
    className={}
    style={}
    electionOM={[electionObj,electionMethods]}
>
```
After someone clicks on Create New, or when they select an existing election they will get to this page:
The Election Header #18 selects which election - will be in a parent component to this one. 

This component will render the Navigation Panel, and the Component being worked on, and allow the navigation.
The Navigation Panel #16 shows the election status, and allows the user to select the component to work with on the right.
The required  components are:
- #44 
- #45 
- #48
- #49 
- #50 
- #51 
- #53 
- #56 
- #57 

- [x] When the user click on an item in the navigation bar, the corresponding component will be rendered to the right.
- [x] electionOM will be passed to each child
- [x] an onDone function will be passed to the child, for use by it's submit/next to cause navigation to go to the next component.  It doesn't matter if what this onDone returns - always go to the next component.
- [x] onDone will be passed to the navigation bar, which will return {valid: xxx, value: component-name}. This component will use that value to decide which child to render.

figma: https://www.figma.com/file/yXrcMvg5EIwdLkHHxIdILh/Undebate?node-id=337%3A4104

![image](https://user-images.githubusercontent.com/3317487/131608833-d1cbba92-3cdd-4ced-a586-7be6ef49ab4d.png)

**Comment by ddfridley** [comment]
Requires Navigation Page

**Comment by ddfridley** [comment]
closed in #157
